src/cmd/package.go

```diff
@@ -300,13 +301,15 @@ func bindCreateFlags() {
 	v.SetDefault(V_PKG_CREATE_SBOM_OUTPUT, "")
 	v.SetDefault(V_PKG_CREATE_SKIP_SBOM, false)
 	v.SetDefault(V_PKG_CREATE_MAX_PACKAGE_SIZE, 0)
+	v.SetDefault(V_PKG_CREATE_SIGNING_KEY, "")
```

Comment:
Thoughts on adding the cosign command line as well so that people can more easily make keypairs?

Reply:
Added a `gen-key` tools command to generate the keypairs